dynamic-layers/qt5-layer/recipes-qt/qt5/qtwebengine_git.bbappend

```diff
@@ -0,0 +1,13 @@
+export NINJAFLAGS="-j 32"
```

Comment:
Not related.

Reply:
I have to admit that I didn't understand it completely. I had problems compiling the chromium part but it worked when restricting with this line which we can of course remove if it works for others without it.

Reply:
This is due to memory consumption. The Chromium is a memory hungry beast. 

The right fix is to limit the number of parallel jobs on the qtwebengine recipe but this does not belong to the `.bbappend` fine.